gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-kafka/src/main/java/io/gravitee/plugin/endpoint/kafka/KafkaEndpointConnector.java

```diff
@@ -127,6 +130,14 @@ protected Completable publish(final ExecutionContext ctx) {
         }
     }
 
+    /**
+     * This method is used to add custom configuration for producer client
+     * @param config
+     */
+    protected void addCustomProducerConfig(final Map<String, Object> config) {
+        // Nothing to do here
```

Comment:
Can you please explain the point of adding those 2 empty methods ?

Reply:
It is for adding custom configuration by the advanced endpoint.

Reply:
Oh, I see, got it !
Yea, a little javadoc rephrase would help